README.md

```diff
@@ -54,7 +54,7 @@ Since the default middleware stack uses url\_encoded middleware and default adap
 ```ruby
 conn = Faraday.new(:url => 'http://sushi.com') do |faraday|
   faraday.request  :url_encoded             # form-encode POST params
-  faraday.response :logger                  # log requests to STDOUT
+  faraday.response :logger                  # log requests to $stdout
```

Comment:
Also: I find this example confusing. `faraday.response` logs _requests_ to standard out? Was this a mistake?

Reply:
That's a fair point. Reality is that logger middleware logs both requests and responses, but for some reason has been historically created as a response middleware.
The example description can probably be changed into `# log requests and responses to $stdout`.
However we're going to do some major documentation refactoring with v1.0, so happy to hold this for now and address it later